aws/config.go

```diff
@@ -318,6 +319,7 @@ type AWSClient struct {
 	outpostsconn                        *outposts.Outposts
 	partition                           string
 	personalizeconn                     *personalize.Personalize
+	prometheusconn                      *prometheusservice.PrometheusService
```

Comment:
Match AWS Go SDK naming.

```suggestion
	prometheusserviceconn               *prometheusservice.PrometheusService
```

Reply:
done.